script/deploy-canary-demo

```diff
@@ -12,10 +12,12 @@ echo "Deploying to alpha stage for canary"
 npm run deploy -- -b chime-sdk-demo-canary -s chime-sdk-demo-canary
 npm run deploy -- -b chime-sdk-meeting-readiness-checker-dev-canary -s chime-sdk-meeting-readiness-checker-dev-canary -a meetingReadinessChecker
 
-
-echo "Deploying to gamma stage for canary"
-
+echo "Deploying to devo stage (that talks to gamma Chime endpoint) for canary"
 export AWS_ACCESS_KEY_ID=$GAMMA_AWS_ACCESS_KEY_ID
```

Comment:
This is not correct right? Both of this deploy to the same aws account (chime-sdk-gamma-canary) and not the devo stage.

Reply:
+1 Remove stage mentions

Reply:
Sorry if the wording was not accurate. I meant the to-be-added devo stage in our canary pipeline. Rephrased a little bit.

Reply:
Discussed offline. We will create a new AWS account for the devo (gamma-endpoint) canary. Will update the PR accordingly.